### DISCUSSION TITLE
Web server cannot connect to itself

### DISCUSSION THREAD

**Discussion by benr77** [nextcloud/all-in-one#4966] [Questions] [answered]
### Steps to reproduce
1. Install a brand new installation of AIO from Digital Ocean Marketplace. Version 29.0.3
2. After installing all the containers, log in and see setup checks or run `occ setupchecks`
3. See lots of errors compaining with message _"To allow this check to run you have to make sure that your webserver can connect to itself. Therefor it must be able to resolve and connect to at least one its `trusted_domains` or the `overwrite.cli.url`."_

### Expected behavior 

Setup checks on a brand new installation should all pass

### Actual behavior 

I think this is because the config checks are now running on the server instead of the client browser, and the Docker configuration is not setup to allow it to see itself outside of the container

### Host OS 

Ubuntu

#### Nextcloud AIO version 

v9.1.0

#### Current channel 

Latest

**Comment by szaimen** [comment, marked as answer]
Hi, I fear I cannot reproduce this on my test server. Do you maybe run a firewall or use cloudflare that makes this fail?

**Comment by styleshady** [comment]
i have the same errros. My setup is behind a traefik-proxy
